CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Mobile Center SDK for iOS Change Log
 
+## Version 0.14.1
+
+This version contains an improvement and a bug fix.
+
+### MobileCenter
+
+* **[Improvement]** The SDK extends background time when the application is backgrounded as a last chance to send any remaining logs.
```

Comment:
I'd say "The SDK requests background time when the app moves into the background to make sure remaining logs are sent. Previously, remaining logs would be sent on the next start up."